include/lbann/utils/distconv.hpp

```diff
@@ -40,6 +40,13 @@
 
 #define DISTCONV_HAS_CUDNN
 
+#if LBANN_DISTCONV_NUM_DIMS == 5
+#define DISTCONV_HAS_DEPTH
```

Comment:
Please rename this to `LBANN_DISTCONV_HAS_DEPTH`. Macro and environment variable names starting with `DISTCONV_` should be reserved for the Distconv package itself. This may not be enforced in the existing code, though.